tofu/geom/_GG.pyx

```diff
@@ -3129,7 +3128,7 @@ def LOS_calc_signal(func, double[:,::1] ray_orig, double[:,::1] ray_vdir, res,
                                                                 ray_orig[:, ii:ii+1],
                                                                 ray_vdir[:, ii:ii+1])
                     val_2d = func(pts, t=t, vect=-usbis, **fkwdargs)
-                    sig_mv[:, ii] = np.sum(val_2d, axis=-1)*loc_eff_res[0]
+                    sig[:, ii] = np.sum(val_2d, axis=-1)*loc_eff_res[0]
```

Comment:
@Didou09 :
> Could you find the reason why an error was thrown with vect=None ? It should not

It doesn't anymore. Here is the only real change in this PR, that fixes the bug.

Reply:
PS: if you see line 3319, it only happened if `ani=True, minimize=hybrid, method=sum`, and if you see line 2863 I took out the variable which was no longer need and verified all loops to see that this error was nowhere else.